Catappult/appcoins-wallet-android
# [WAL-111] Eternal loading after block on transfer

**What does this PR do?**

   Fixes a bug where after showing the blocked warning and user dismissed it, the user would continue to see the loading screen.  This was made in the first commit
  The second commit aims to move the TransferFragment to the new architecture.

**Database changed?**

No

**Where should the reviewer start?**

TransferPresenter.kt

**How should this be manually tested?**

For the bugfix, use a blocked wallet and trying to transfer credits, after getting the dialog dismiss it and check if the loading screen doesn't appear.
Try to read a QR Code. And make a transfer of APPC, Credits and ETH (low values :))

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/WAL-111

**Code Review Checklist**
